sunpy/map/tests/test_map_factory.py

```diff
@@ -243,6 +243,13 @@ def test_url_pattern():
     amap = sunpy.map.Map("http://data.sunpy.org/sample-data/AIA20110319_105400_0171.fits")
     assert isinstance(amap, sunpy.map.GenericMap)
 
+@pytest.mark.remote_data
+def test_uri_pattern():
+    """
+    Testing publicly accessible s3 object
+    """
+    amap = sunpy.map.Map("s3://gov-nasa-hdrl-data1/sdo/aia/20100513/0131/sdo_aia_h2_20100513T000400_0131_v1.fits", fsspec_kwargs={"anon": True})
```

Comment:
Do you happen to know why `anon=True` needs to be specified? it seems like a sane default to me?

Reply:
`anon=True` is set so that the file will be read from AWS S3 without using AWS keys and only works if the file is from a public repository. I set it like this in the test scripts so that I don't have to inject AWS permissions into the test environment.

Most likely it's fine to set `anon=True` as default because most science repos are public, but it seems a bit backwards to me since it's normally not how the typical user would interact with AWS. For example if someone was trying to access files from a personal s3 repo (usually not public), `anon=True` would break.